tools/xtce_generation/ccsds-header.xml

```diff
@@ -45,7 +45,7 @@
 			<xtce:Parameter name="PKT_APID" parameterTypeRef="uint11">
```

Comment:
What is this file used for?

Reply:
This was just an example of the ccsds header only. It can totally be removed because it is not used for anything, just an example. Its the output of `generate_ccsds_header_xml.py` which is also outdated I think.

Reply:
I'd say remove it if it's not used